assets/compat.ini

```diff
@@ -804,6 +804,17 @@ ULUS10132 = true
 # Homebrew Kurok (we don't normally add homebrew here but it has a distinctive GameID)
 KURO00767 = true
 
+# Scooby-Doo! Who's Watching Who?
+ULUS10168 = true
+ULES00571 = true
+
+# Tron Evolution
+ULUS10548 = true
+ULES01494 = true

```

Comment:
There's also ULES01495, see https://report.ppsspp.org/games?name=tron and hover the game names